src/Symfony/Component/Security/Http/Firewall/ContextListener.php

```diff
@@ -198,7 +220,7 @@ protected function refreshUser(TokenInterface $token)
         }
 
         if ($userNotFoundByProvider) {
-            return;
+            return null;
```

Comment:
To be reverted as it should be applied to the 2.7 branch. But it probably worth a dedicated PR on the whole codebase. Related to #17201.

Reply:
Will fix